src/control_flow/test_for.py

```diff
@@ -51,20 +51,23 @@ def test_for_statement():
     # To iterate over the indices of a sequence, you can combine range() and len() as follows:
     words = ['Mary', 'had', 'a', 'little', 'lamb']
     concatenated_string = ''
+    concatenated_string2 = ' '.join(words)
```

Comment:
This was an example for testing `for` control, so let's add an example for that.